orchestrator/db/filters/generic_filters/bool_filter.py

```diff
@@ -13,12 +13,12 @@
 
 from typing import Callable
 
-from sqlalchemy import Column
+from sqlalchemy import ColumnClause
 
 from orchestrator.db.filters.filters import QueryType
 
 
-def generic_bool_filter(field: Column) -> Callable[[QueryType, str], QueryType]:
+def generic_bool_filter(field: ColumnClause) -> Callable[[QueryType, str], QueryType]:
     def bool_filter(query: QueryType, value: str) -> QueryType:
         value_as_bool = value.lower() in ("yes", "y", "true", "1", "ja", "j")
```

Comment:
I vaguely remember having seen a function for this str -> bool conversion

Reply:
YAML?

Reply:
I opt for the first one :sweat_smile: